### PR TITLE
update to api-common 1.8.1

### PR DESCRIPTION
**Pull request by elharo**
@andreamlin

### CODE REVIEW

File: google-cloud-clients/pom.xml [hunk 1]
```diff
@@ -157,14 +157,14 @@
...
-    <api.common.java.version>1.7.0</api.common.java.version>
+    <api.common.java.version>1.8.0</api.common.java.version>
     <google.auth.version>0.15.0</google.auth.version>
     <grpc.version>1.19.0</grpc.version>
     <!-- We are currently using the *-android version to support JDK7. -->
     <guava.version>27.1-android</guava.version>
     <http-client-bom.version>1.29.0</http-client-bom.version>
     <oauth-client.version>1.28.0</oauth-client.version>
-    <protobuf.version>3.7.0</protobuf.version>
+    <protobuf.version>3.7.1</protobuf.version>
```
Comment (sduskis) [inline google-cloud-clients/pom.xml]: Interesting.. grpc is still on 3.6.1 as of their 1.19.0 release.  grpc 1.20.0 upgraded protobuf to 3.7.1.  I think that we ought to upgrade grpc as well, and probably opencensus to 0.19.2

Comment (elharo) [reply google-cloud-clients/pom.xml]: Yes, that makes sense. Not sure any of this is going to work though. There's a dependency-convergence error in google-cloud-core. Probably have to push api-common 1.8.1 and try again to fix that.

Comment (andreamlin) [reply google-cloud-clients/pom.xml]: 1.8.1 is out for api-common-java

Comment (elharo) [reply google-cloud-clients/pom.xml]: upgraded to 1.8.1; testing now

Comment (elharo) [reply google-cloud-clients/pom.xml]: [ERROR] Errors: 
[ERROR] com.google.cloud.datastore.DatastoreTest.com.google.cloud.datastore.DatastoreTest
[ERROR]   Run 1: DatastoreTest.beforeClass:157 » IllegalArgument Invalid version format
[ERROR]   Run 2: DatastoreTest.afterClass:179 » Connect Connection refused (Connection refused)
[INFO] 
[ERROR]   LocalDatastoreHelperTest.testStartStopReset:94 » IllegalArgument Invalid versi...
[INFO] 
[ERROR] Tests run: 168, Failures: 0, Errors: 2, Skipped: 0
[INFO]

**Comment by chingor13** [comment]
> api-common 1.8.1 does seem to work here.

For convergence or for tests? It looks like CI passed for the unit tests.

**Comment by chingor13** [comment]
Oops, I misread your comment as it didn't pass.

**Pull request merged**